lib/mailtrap/sending/convert.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module Mailtrap
+  module Sending
+    module Convert
+      class << self
+        def from_message(message) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
Should we commit code with `rubocop:disasble`?

Reply:
I don't think it's a good idea to liberate rubocop settings for the entire repo having in mind we are talking about a single method with disables.
Any ideas to simplify the code?